generator/index.js

```diff
@@ -35,10 +35,10 @@ const targets = [{
 }];
 
 /**
- * Emoji codepoints to globally ignore.
+ * Emoji codepoints which are duplicates. These are marked as such in the generated code.
  * @type {string[]}
  */
-const ignore = ["1F926", "1F937", "1F938", "1F93C", "1F93D", "1F93E", "1F939"];
+const duplicates = ["1F926", "1F937", "1F938", "1F93C", "1F93D", "1F93E", "1F939"];
```

Comment:
Do the duplicates have different information in the original datasource or what? Is it worth filling bugs at the datasource repo?

Reply:
Thanks for the explanation.